.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Dispatch on Label
```

Comment:
Not exactly sure what the name is supposed to mean, would you mind changing that to `build`?

Reply:
Sure.